scripts/enable

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+set -e
+
+if [ -z $SUDO_USER ]; then
+    echo "Run this command with sudo"
+    exit 1
+fi
+
+mkdir -p /etc/filebeat
+cp docker/filebeat.yml /etc/filebeat
```

Comment:
the idea here is that we'll end up with other configurations most likely and eventually this script will take arguments